packages/core/src/spawn.ts

```diff
@@ -123,5 +138,5 @@ export function createSpawner(
       actorRef.start();
     });
     return actorRef;
-  };
+  }) as Spawner<any>;
```

Comment:
The return type errors without this bypass. I think this is expected given that typescript has not enough information to correctly type-check all possible configurations.

All tests are passing, so I think it shouldn’t be an issue.